Editor/Scripts/wind_forces_tutorial.py

```diff
@@ -0,0 +1,94 @@
+        self.add_step(TutorialStep("Add a Tag component",  """<html><p style="font-size:13px">
+                We need to add a tag to the entity to identify it as a wind provider and specify the type of wind force. We can do this with a Tag component.
+                In Entity Inspector, choose Add Component and select Tag from the components list to add a Tag component to the entity. </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Create a local wind force tag",  """<html><p style="font-size:13px">A wind provider can create a global wind force for the entire level, or a local wind force that is contained within a volume. Let's create a local wind force.
+                In the Tag component, click the + button to add a new tag element. Enter wind in the new element field to create a local wind force.<br><br> </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Add a tag component for the entity",  """<html><p style="font-size:13px"> Edit the tag value to <b>global_wind</b>.
+                <br><br> There are two types of wind: global and local, used to specify which entities provide wind forces. <br><br>
+                If you choose to use the Local wind tag property, the wind force affects only certain entities.</p></html>""", "EntityOutlinerWidgetUI"))
+        self.add_step(TutorialStep("Add a PhysX collider component",  """<html><p style="font-size:13px"> Wind providers must have a PhysX collider. For local wind forces, the collider specifies the volume that contains the wind force. 
+                Objects are affected by the local wind force only when they are inside the wind provider's PhysX collider.
+                In Entity Inspector, choose Add Component and add a PhysX Collider component to the entity. <br><br> </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Set the collider shape", """<html><p style="font-size:13px">We'll use a simple box for the volume of our local wind force.<br><br>
+                In <b>Entity Inspector</b>, in the <b>PhysX Collider</b> component, set the <b>Shape</b> property to 
+                <strong><code style="font-size:14px;color:#E44C9A">"Box"</code></strong>.<br><br>
+                The wind force can only affect entities that are within this box collider shape. </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Resize the wind provider", """<html><p style="font-size:13px">The box is quite small. Let's enlarge it so that it affects a larger area and so that we can easily position the cloth entity inside it.
+                <br><br>In <b>Entity Inspector</b> in the <b>PhysX Collider</b> component, set the <b>X</b>, <b>Y</b>, and <b>Z</b> components of the <b>Dimensions</b> property to 
+                <strong><code style="font-size:14px;color:#E44C9A">"text"</code></strong>.</p></html>""", "InspectorMainWindow"))
```

Comment:
Highlight the Dimensions property

```suggestion
        self.add_step(TutorialStep("Resize the wind provider", """<html><p 
                style="font-size:13px">The box is quite small. Enlarge it 
                so that it affects a larger area and so that you can easily 
                position the cloth entity inside it.<br><br>In <b>Entity 
                Inspector</b>, in the <b>PhysX Collider</b> component, set the 
                <b>X</b>, <b>Y</b>, and <b>Z</b> components of the 
                <b>Dimensions</b> property to <strong><code 
                style="font-size:14px;color:#E44C9A">5.0</code></strong>.</p>
                </html>""", {"text": "Dimensions", "type": QtWidgets.QFrame}))
```